code/modules/mob/living/carbon/human/species_types/jellypeople.dm

```diff
@@ -182,11 +183,12 @@
 	background_icon_state = "bg_alien"
 
 /datum/action/innate/split_body/IsAvailable()
-	if(..())
-		var/mob/living/carbon/human/H = owner
-		if(H.blood_volume >= BLOOD_VOLUME_SLIME_SPLIT)
-			return 1
-		return 0
+	. = ..()
+	if(!.)
+		return
+	var/mob/living/carbon/human/H = owner
+	if(!(H.blood_volume >= BLOOD_VOLUME_SLIME_SPLIT))
+		return FALSE
```

Comment:
readability, you could also keep the if(..()) to avoid having to return FALSE at the end
```suggestion
	if(H.blood_volume >= BLOOD_VOLUME_SLIME_SPLIT)
		return TRUE
	return FALSE
```